src/status_im/contexts/chat/messenger/messages/list/view.cljs

```diff
@@ -379,28 +187,21 @@
     (reset! distance-atom new-distance)))
 
 (defn messages-list-content
-  [{:keys [insets distance-from-list-top content-height layout-height distance-atom
-           chat-screen-layout-calculations-complete? chat-list-scroll-y]}]
-  (let [theme                                (quo.theme/use-theme)
-        {:keys [chat-type chat-id] :as chat} (rf/sub [:chats/current-chat-chat-view])
-        one-to-one-chat?                     (= chat-type constants/one-to-one-chat-type)
-        community-channel?                   (= constants/community-chat-type chat-type)
-        {contact-customization-color
-         :customization-color}               (when one-to-one-chat?
-                                               (rf/sub [:contacts/contact-by-identity chat-id]))
-        customization-color                  (cond community-channel?
-                                                   (or (:color chat)
-                                                       (rf/sub [:communities/community-color
-                                                                (:community-id chat)]))
-                                                   one-to-one-chat? contact-customization-color
-                                                   :else (or (:color chat) :turquoise))
-        {:keys [keyboard-shown]}             (hooks/use-keyboard)
-        {window-height :height}              (rn/get-window)
-        context                              (rf/sub [:chats/current-chat-message-list-view-context])
-        able-to-send-message?                (:able-to-send-message? context)
-        messages                             (rf/sub [:chats/raw-chat-messages-stream chat-id])
-        margin-bottom?                       (and community-channel? (not able-to-send-message?))
-        recording?                           (rf/sub [:chats/recording?])]
+  [{:keys [insets distance-from-list-top layout-height chat-list-scroll-y on-layout-done?]}]
+  (let [content-height           (rn/use-ref-atom 0)
+        distance-atom            (rn/use-ref-atom 0)
```

Comment:
These two values were received, but there's no need to lift up the state even more, they are only used here, so they are being created here.